scg-system/src/test/files/yaml/config-abac-multiple.yaml

```diff
@@ -1,27 +1,30 @@
 version: "1.0"
 server:
   name: "Fuseki server simple"
+prefixes:
+  - prefix: "authz"
+    namespace: "http://telicent.io/security#"
```

Comment:
Also based on our discussions yesterday I changed this config to explicitly specify the `authz` operations so that your ABAC test case that uses this config passes as intended